portfolio/src/main/webapp/script.js

```diff
@@ -45,18 +45,22 @@ function showProjectDetail(event) {
   targetElement.parentElement.appendChild(descriptionDiv);
 }
 
+Window.onload = showComments();
```

Comment:
The W in window should be small right?

Reply:
Both of them work. I have changed it to lower case for consistency.